data/presets/presets/tourism/artwork/graffiti.json

```diff
@@ -0,0 +1,26 @@
+{
+    "icon": "maki-art-gallery",
+    "fields": [
+        "name",
+        "artist"
+    ],
+    "geometry": [
+        "point",
+        "vertex",
+        "line",
+        "area"
+    ],
+    "tags": {
+        "tourism": "artwork",
+        "artwork_type": "graffiti"
+    },
+    "reference": {
+        "key": "artwork_type"
+    },
+    "terms": [
+        "graffiti",
+        "Street Artwork",
+        "Guerilla Artwork"
+    ],
+    "name": "Graffiti Artwork"
```

Comment:
Can we just call it Graffiti?

Reply:
Maybe... the problem is, that not all graffiti is art. Actually, most of graffiti is not art, but just juvenile scribbling. This wouldn't and shouldn't be tagged with artwork. Perhaps if it is called just "Graffiti", people might get the wrong idea about what graffiti should be mapped.

An uncertainty that could certainly be solved with your subtitles.

Reply:
I'm actually fine with any graffiti being tagged this way. If we specify this as only for "art" graffiti then we'll have people arguing over what's art and what's not art. A statue is a statue based on its form, not its quality. I think we can treat this the same.

Reply:
What is art and what is not art will always be subjective. But this is nothing iD contributors need to deal with.

Fact is, the tag is `tourism=artwork`. As iD does not expose the tag names (immediately) to contributors, this information - that it is about art - would be lost if it is not mentioned somehow in the name.

[This, for example, is (also) graffiti.](https://proxy.duckduckgo.com/iu/?u=http%3A%2F%2F1.bp.blogspot.com%2F_VezUbgtAF0c%2FTAjzJ7hbMzI%2FAAAAAAAACzg%2FBhhc0mcPss4%2Fs1600%2Fgraffiti_tags.jpg&f=1) The most common form I see around my area, actually.

An (art)work of graffiti is something very different from that.

Reply:
> An (art)work of graffiti is something very different from that.

I guess we just disagree on this point. To me graffiti is graffiti. This seems like a silly thing to argue about so I suggest we just hold off on adding this preset until we add the subtitle property.

Reply:
I may be totally wrong on this, but I believe there's a weird ambiguous legal distinction (in many places) between what is called "street art" and what is called "graffiti". As far as I can tell, they're the same thing except the former is deemed to have some worth. Quite an interesting topic to read up on.

Reply:
Dont we have the same issue with other presets and tags that have subjective importance? Like trees. First only very important trees where added. Now there are a lot of them on roads and such. And new use cases and data explorations come out of them. Some renderer still complain that people add too many trees which they need to filter; some projects are happy to calculate the green-ness of a hood. IMO mapper will use their own judgement here just fine; it happened before :).

Naming: I suggest keeping the actual name short but checking the info-(i) description and maybe make sure that it hints at artworks, not at scribbles/tagging (which will be pulled from Wikibase ATM, right?).

Reply:
Maybe my comments made it appear I have a stronger opinion about it that I actually do. I just wanted to show that I gave thought to the name choice.
Just "Graffiti" is fine for me too, especially since it's still the artwork icon and the tags are as always still visible further below.

Reply:
Thanks for the thoughtful discussion, everyone. Apologies if I came across as grumpy.